src/logistration/LoginPage.jsx

```diff
@@ -104,19 +104,29 @@ class LoginPage extends React.Component {
   }
 
   render() {
+    const { currentProvider, finishAuthUrl, providers } = this.props.thirdPartyAuthContext;
+
     return (
       <>
         <RedirectLogistration
           success={this.props.loginResult.success}
           redirectUrl={this.props.loginResult.redirectUrl}
+          finishAuthUrl={finishAuthUrl}
```

Comment:
I think redirection won't happen for finishAuthUrl because it depends upon the success value and success value would be always false in your case.